cylc/flow/tui/__init__.py

```diff
@@ -89,7 +89,10 @@
 TASK_MODIFIERS = {
     'held': '\u030E',
     'queued': '\u033F',
-    'runahead': '\u030A'
+    'runahead': '\u0340',
+    'xtriggered': '\u1DC9',
+    'wallclock': '\u135D',
```

Comment:
I wonder if these two chars might be more intuitive...
```suggestion
    'xtriggered': '\u2DEF',
    'wallclock': '\u20D9',
```

Reply:
* 0X2DEF - too small -   ⷯO
* 0X20D9 - too small and wrong placement -  ⃙O

Reply:
* 0X036C - too small
* 0X1AB2 - wrong placement
* 0X1AB3 - too small and wrong placement